resources/quality/z-bolt/bowden/ABS/volcano 0.6/z-bolt_bowden_volcano_0.60_ABS_sprint1.inst.cfg

```diff
@@ -0,0 +1,23 @@
+[general]
+version = 4
+name = Sprint [1]
```

Comment:
You might want to give these profiles different names than "various numbers of sprint" or re-weight the user-visible names if the Draft profile isn't really intended for quick drafts.